core/labeled-replies-3.3.md

```diff
@@ -38,7 +38,7 @@ Labeled responses allow a much simpler way to correlate using a single id attach
 
 ### Dependencies
 
-This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and relies on support for the [`batch`](/specs/extensions/batch-3.2.html) capability.
+This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and relies on support for the [`batch`](/specs/extensions/batch-3.2.html) capability which MUST be negotiated at the same time.
```

Comment:
The `draft/label` tag can be useful in the absence of `batch`. While batches are useful and can aid in keeping relevant content together, it should not be required for all uses of the `draft/labeled-response` message tag. There are instances where it may be necessary to uniquely identify a labeled response for content that does not belong to a batch.

See ircv3/ircv3-ideas#6